We gratefully acknowledge the authors, originating and submitting laboratories of the genetic sequences and metadata for sharing their work. Please note that although data generators have generously shared data in an open fashion, that does not mean there should be free license to publish on this data. Data generators should be cited where possible and collaborations should be sought in some circumstances. Please try to avoid scooping someone else's work. Reach out if uncertain.


#### Analysis
Our bioinformatic processing workflow can be found at [github.com/nextstrain/rabies](https://github.com/nextstrain/rabies) and includes:
- sequence alignment by [augur align](https://docs.nextstrain.org/projects/augur/en/stable/usage/cli/align.html)
- phylogenetic reconstruction using [IQTREE-2](http://www.iqtree.org/)
- ancestral state reconstruction and temporal inference using [TreeTime](https://github.com/neherlab/treetime)

#### Underlying data
We curate sequence data and metadata from NCBI as starting point for our analyses. Curated sequences and metadata are available as flat files at:
- [data.nextstrain.org/files/workflows/rabies/sequences.fasta.zst](https://data.nextstrain.org/files/workflows/rabies/sequences.fasta.zst)
- [data.nextstrain.org/files/workflows/rabies/metadata.tsv.zst](https://data.nextstrain.org/files/workflows/rabies/metadata.tsv.zst)

#### Clade annotation

Clades are defined based on the "clades", "subclades", and "lineages" classified in [Campbell et al. 2022](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1010023), which draws from the classifications of [Troupin et al. 2016](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1006041).

---

Screenshots may be used under a [CC-BY-4.0 license](https://creativecommons.org/licenses/by/4.0/) and attribution to nextstrain.org must be provided.
